index.bnd

```diff
@@ -7,4 +7,7 @@
         description="Use the maven directory layout for sources and binaries", \
     bndtools/workspace-templates/osgi; \
         name=osgi; \
-        description="OSGi R8 with Felix distribution"
\ No newline at end of file
+        description="OSGi R8 with Felix distribution", \
+    mnlipp/de.mnl.osgi/de.mnl.osgi.bnd.repository/workspace-template; \
```

Comment:
Thanks for your contribution @mnlipp 

```suggestion
    mnlipp/de.mnl.osgi/tree/b96e0a8877bad1c68cdc050d5854829253ef63bb/workspace-template; \
```

Could you please change it to this, to pin it to this specific commit SHA? (points to [here](https://github.com/mnlipp/de.mnl.osgi/tree/b96e0a8877bad1c68cdc050d5854829253ef63bb/de.mnl.osgi.bnd.repository/workspace-template))

This way we make sure that the template fragment which is selected and used by the user in Eclipse is always in a predictable state of this PR.

Reply:
Of course, I can do this. The question is, whether this is the best approach for the "bndtools/workspace-templates" project. Contrary to the "built-in" samples, my template references a versioned jar. This implies that every time I upgrade the plugin, a new PR is required here, creating a (limited) workload for the maintainers (you, obviously). This might apply to other future contributions as well.

Reply:
You are right in your observation. But we also need some kind of predictability for "3rd Party" templates. This means, that a user should always get the same template which passed the "code review".  At least for the default behavior.

But I see your point and I think there is room for discussion how to handle this situation going forward. 
Maybe we could modify the UI to allow the user to "force" the master/main branch. But this would need discussion.  
At the moment there is this little "+" button where you can enter a custom URL. 

I would take this into our [next bnd call](https://bnd.discourse.group/t/you-wanna-be-a-bnd-developer/443) where you are very welcome to join. Additionally we could create an [issue in bnd](https://github.com/bndtools/bnd/issues) where we discuss this.